### PR TITLE
Climber integration

### PR DESCRIPTION
**Pull request by wakandacat**
changes to climber to make it so the trigger on the controller adjusts speed

### CODE REVIEW

File: src/org/usfirst/frc/team2706/robot/subsystems/Climber.java [hunk 1]
```diff
@@ -109,9 +109,27 @@ public void initDefaultCommand() {}
...
+        if (speed <= 0) {
+            speed = 0;
+        } else if (speed >= 1) {
+            speed = 1;
+        }
+        motor.set(speed);
+        debugOutput();
+    }
+    
+    public void debugOutput() {
```
Comment (ryanlarkin) [inline src/org/usfirst/frc/team2706/robot/subsystems/Climber.java:129]: Can you remove calls to this method

**Review by ryanlarkin — CHANGES_REQUESTED** [review]
Just fix the debug printing, and it looks good

File: src/org/usfirst/frc/team2706/robot/subsystems/Climber.java (hunk 1 above)
Comment (wakandacat) [reply src/org/usfirst/frc/team2706/robot/subsystems/Climber.java:129]: we can't remove it yet because we are not done debugging

Comment (ryanlarkin) [reply src/org/usfirst/frc/team2706/robot/subsystems/Climber.java:129]: Ok, I'm going to wait to merge it until you are done

**Review by ryanlarkin — APPROVED** [review]
I'm going to merge this so drive team can use it, but the debugPrints should be removed by the time we get to competition

**Pull request merged**